### PR TITLE
finished HW5

### PR DESCRIPTION
**Pull request by shaibrown5**

### CODE REVIEW

File: HW5/server.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const fs = require('fs');
+let app = express();
+app.use(express.static('./files'));
+
+
+app.get('/server/:dir/:fileName',(req,res) => {
+	let dirName = req.params.dir || null;
```
Comment (talboren) [inline HW5/server.js:9]: what for?

File: HW5/server.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	// conditional statements to handle illegal urls
+	if(!dirName){
+		res.send("invalid dir name!");
+	}
+	else if(!fileName){
+		res.send("No file was requested! please enter a file name");
+	}
+	
+	// concat dir name and file name into a single path
+	let path = "./" + dirName + "/" + fileName;
```
Comment (talboren) [inline HW5/server.js:21]: This is a security fault, read about "path traversal" (just a point to learn from :))

**Review by talboren — APPROVED** [review]
great overall job.

File: HW5/server.js (hunk 2 above)
Comment (shaibrown5) [reply HW5/server.js:21]: so how would you go about making the path, read a bit about path traversal

**Pull request merged**